Assets/Scripts/Event Manager.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+
+[CreateAssetMenu(fileName = "EventManager", menuName = "Scriptable Objects/EventManager")]
+public class EventManager : ScriptableObject
+
+{
+    public static EventManager instance;
+
+    public delegate void ButtonClickEvents();
+    public ButtonClickEvents onForwardClick;
+    public ButtonClickEvents onBackwardClick;
+
+
+    public delegate void SceneEvent();
+    public SceneEvent onOpenGame;
+
+    public void Initialise()
+    {
+        instance = this;
+    }
+
+    public void HandleForwardClick()
```

Comment:
This should probably be called TriggerForwardClick() and not Handle, as this function is going to Start the event, not subscribe. The term "HandleX" is used to indicate that this function is *intended* to be called as a response from event X being triggered.

Reply:
Resolved